### ISSUE TITLE
Add a New Field Named 'From Another University' in Doctype: 'Faculty Member' and Link it to a New Doctype Named 'University'.

### ISSUE THREAD

**Issue by EsmailEbrahim** [Sana-a-Uni/Academia#57]
In Doctype: 'Faculty Member', Add a New Field Named 'From Another University' with Display Depends on the Field that Shows if it is Contract then Link it to a New Doctype Named 'University' with Fields: 'University Name' (Data, Required), 'Country' (Link->Country, Required), and 'City' (Data).

@ju-omari21 
@alshalabi-su

**Comment by Ali-Alnosairi** [comment]
For what purpose you need these additions.
I think there is no University doctype anymore because we use the company doctype instead. 

> In Doctype: 'Faculty Member', Add a New Field Named 'From Another University' with Display Depends on the Field that Shows if it is Contract then Link it to a New Doctype Named 'University' with Fields: 'University Name' (Data, Required), 'Country' (Link->Country, Required), and 'City' (Data).
>

**Comment by EsmailEbrahim** [comment]
There is a need in our side, we need to know if a faculty member is
contracted. If it is contracted there is need to know if he is from another
university, so we want to keep the universities in a list to apply our
filters when choosing 'Discussion Committe'.

**Comment by ju-omari21** [comment]
But as the Dean said, every university is a company, so there is no need to the university doctype.

**Comment by ju-omari21** [comment]
@EsmailEbrahim 
I am working with the additions you need. So, the feature you want is to know if the faculty member is contracted to another university or not, right?

**Comment by EsmailEbrahim** [comment]
Yes, it is to know if the faculty member is contracted, and if he is from another university what is the university.

So, I think in Faculty Member there should be three fields for that purpose:
1- Is Contracted(Check).
2- Contract Status(Select, Display Depends on -> Is Contracted).
3- University(Link -> University, Display Depends on -> Is Contracted).

@ju-omari21

**Comment by ju-omari21** [comment]
@EsmailEbrahim 
Got you, there is now a field "Contract Status: Type == data" in the faculty member, and the dean said that this field will be fetched from the employee in HR, but we have not customized this yet.

**Comment by EsmailEbrahim** [comment]
Contract Status currently it is ok, you can add the two other fields until customization be ready with HRMS, so there is no problem with that, thanks 👍

@ju-omari21
